test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -1268,6 +1267,11 @@ public final void testSyntheticEmptyList() throws IOException {
         assertThat(syntheticSource(mapper, b -> b.startArray("field").endArray()), equalTo(expected));
     }
 
+    private boolean shouldUseIgnoreMalformed() {
+        // 5% of test runs use ignore_malformed
+        return supportsIgnoreMalformed() && randomDouble() >= 0.05;
```

Comment:
should be `<=` instead?

Reply:
Wow, of course. I guess it's a good stress test to run this on CI :)